android/libraries/rib-router-navigator/src/main/kotlin/com/uber/rib/core/RouterNavigator.kt

```diff
@@ -95,18 +94,16 @@ interface RouterNavigator<StateT : RouterNavigatorState> {
 
   /**
    * Switch to a new state - this will switch out children if the state is not the current active
-   * state already. The transition will be controlled by the [StackRouterNavigator.Flag]
-   * provided.
-   *
+   * state already. The transition will be controlled by the [StackRouterNavigator.Flag] provided.
    *
    * NOTE: This will retain the Riblet in memory until it is popped or detached by a push with
    * certain flags.
    *
    * @param newState to switch to.
    * @param attachTransition method to attach child router.
    * @param detachTransition method to clean up child router when removed.
-   * @param <R> router type to detach.
-   </R> */
+   * @param <R> router type to detach. </R>
```

Comment:
I'd remove those `@param` that add nothing to the reader. I'd KDoc it like the following:
```kotlin
/**
 * Detaches the current state using [detachTransition] and attaches [newState] using [attachTransition]. 
 * The resulting stack depends on the [flag].
 */
```

Reply:
I will do it in separate PR, please check newest version of PR, it doesn't include changes like this one